src/eureka/S1_detector_processing/bias_sub.py

```diff
@@ -0,0 +1,187 @@
+# Module for subtracting a super-bias image from science data sets
+# This is based on the superbias_step from the JWST pipeline v1.8.0
+# adapted by Kevin Stevenson, Feb 2023
+
+import numpy as np
+import logging
+from jwst.lib import reffile_utils
+from .group_level import mask_trace
+from ..lib.astropytable import savetable_S1
+from ..lib.smooth import smooth
+
+log = logging.getLogger(__name__)
+log.setLevel(logging.DEBUG)
+
+
+def do_correction(input_model, bias_model, meta, log):
+    """
+    Short Summary
+    -------------
+    Execute all tasks for Super-Bias Subtraction
+
+    Parameters
+    ----------
+    input_model: data model object
+        science data to be corrected
+
+    bias_model: super-bias model object
+        bias data
+
+    Returns
+    -------
+    output_model: data model object
+        bias-subtracted science data
+
+    """
+    # Check for subarray mode and extract subarray from the
+    # bias reference data if necessary
+    if not reffile_utils.ref_matches_sci(input_model, bias_model):
+        bias_model = reffile_utils.get_subarray_model(input_model, bias_model)
+
+    # Compute trace mask, where ones are good background regions
+    input_model = mask_trace(input_model, log, meta)
+    trace_mask = input_model.trace_mask
+
+    # Compute median of bias frame outside of trace
+    bias_median = np.nanmedian(bias_model.data[np.where(trace_mask)])
+
+    log.writelog('  Computing scale factor for superbias correction.')
```

Comment:
Do we want to force all of these logs to be verbose, or should we allow the prints to the console to be suppressed if `not meta.verbose`

Reply:
Yes!  Unlike a lot of Stage 1 output, this is actually helpful.